os/include/queue.h

```diff
@@ -77,6 +77,26 @@
 #define sq_init(q) do { (q)->head = NULL; (q)->tail = NULL; } while (0)
 #define dq_init(q) do { (q)->head = NULL; (q)->tail = NULL; } while (0)
 
+#define sq_move(q1,q2) \
+  do \
+    { \
+      (q2)->head = (q1)->head; \
+      (q2)->tail = (q1)->tail; \
+      (q1)->head = NULL; \
+      (q1)->tail = NULL; \
+    } \
+  while (0)
```

Comment:
Are you able to modify the coding rule later together??
You modified huge amount.

Reply:
Yes. We will refer back to this PR and make all the coding rule changes in a new PR.